src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -179,6 +179,12 @@ export default class Resources extends Module {
 	}
 
 	checkBloodOvercap(actionBloodChange) {
+		if (actionBloodChange < 0) {
+			// Spender
+			if (this.combatants.selected.hasStatus(STATUSES.DELIRIUM.id)) {
+				actionBloodChange = 0
```

Comment:
_Generally_ avoid modifying function args, pretend they're `const`. In this particular case, the `if` below checks against this value, so you can probably afford to

```suggestion
				return
```

to signify there's no consumption

Reply:
Change updated, also consolidated the double-IF into a single and statement.